docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Messaging_JDBC_Store_for_Messaging_Journal.adoc

```diff
@@ -50,12 +34,29 @@ The name of these tables can be configured with the
 `journal-bindings-table`, `journal-messages-table`,
 `journal-page-store-table, and journal-large-messages-table`.
 
+Please note that the configuration of the underlying pool is something that you need to take care of.
+You  need at least four connections:
+
+ * one for the binding
+ * one for the messages journal
+ * one for the lease lock (if you use HA)
+ * one for the node manager shared state (if you use HA)
+
+So you should define a https://docs.wildfly.org/22/wildscribe/subsystem/datasources/xa-data-source/index.html#attr-min-pool-size[`min-pool-size`] of 4 for the pool. +
+But one fact that you need to take into account is that paging and large messages can use an unbounded number of threads.
+The size, a.k.a. https://docs.wildfly.org/22/wildscribe/subsystem/datasources/xa-data-source/index.html#attr-max-pool-size[`max-pool-size`], of the pool should be defined according to the amount of concurrent threads that perform page/large message streaming operations. There is no defined rule for this as there is no 1-1 relation between the number of threads and the number of connections. The number of connections depend on the number of threads processing paging and large messages operations as well as the time you are willing to wait to get a connection (cf. https://docs.wildfly.org/22/wildscribe/subsystem/datasources/xa-data-source/index.html#attr-blocking-timeout-wait-millis[`blocking-timeout-wait-millis`]). When new large messages or paging operations occur they will be in a dedicated thread and will try to get a connection, being enqueued until one is ready or the time to obtain one runs out which will create a failure. +
+You really need to tailor your configuration according to your needs and test it in your environment following the <<DataSource>> subsystem documentation and perform tests and peroformance runs before going to production.
+
+
 [[reference]]
 == Reference
 
 ****
 
 * Artemis JDBC Persistence -
 https://activemq.apache.org/components/artemis/documentation/latest/persistence.html#configuring-jdbc-persistence
-
+* WildFly DataSource subsystem documentation -
+<<DataSource>>
+* WildFly datasource configuration model
+https://docs.wildfly.org/22/wildscribe/subsystem/datasources/xa-data-source/index.html
```

Comment:
This can be a link that's relative to the current release's docs set. For example:

https://github.com/wildfly/wildfly/blame/master/docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Undertow_using_as_a_Load_Balancer.adoc#L132

Result is at the line above the start of https://docs.wildfly.org/22/Admin_Guide.html#Messaging